src/backend/models/bookingTypes.ts

```diff
@@ -33,26 +33,30 @@ export interface BookingTypeQueryResult {
   name: string;
   created: number;
   rules: BookingTypeDailyRules;
-  exceptions: Array<string>;
+  exceptions: Record<string, never> | Array<string>;
   additional_information?: string | null;
 }
 
 function queryResultToBookingType(row: BookingTypeQueryResult): BookingType {
+  const exceptions =
+    !row.exceptions || Object.keys(row.exceptions).length < 1
+      ? []
```

Comment:
Since `exceptions` type is `json` in the database, passing an empty array will result in an empty object being written to the database instead :sweat_smile: So we'll need to convert any empty object to an empty array here. I suppose updating the db type to something like `json[]` may solve the issue better, but that would require migrating existing data to the new type, which I may only have time to go through later...

Reply:
That makes sense! The one thing I usually recommend is to add that comment as a code comment, so that we remember when reading through :D